<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dignityDialog</class>
 <widget class="QDialog" name="dignityDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agregar/Modificar Dignidades</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="planetaLabel">
       <property name="text">
        <string>Planeta:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="planetaComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="dignidadLabel">
       <property name="text">
        <string>Dignidad:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="dignidadComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
